MIGRATION.md

```diff
@@ -43,7 +43,32 @@ In version 8 of this library, the following components have been added or remove
 ### Removed
 
 - Typography
-    - This component has been removed from the library can be replaced by [Text](https://callstack.github.io/react-native-paper/docs/components/Text/) component from React Native Paper.
+    - This component has been removed from the library can be replaced by [Text](https://callstack.github.io/react-native-paper/docs/components/Text/) component from React Native Paper. 
+    - `Text` component does not support the `color` prop. It must be styled through `style` prop and passing the value from the `theme` object returned by [useExtendedTheme()](https://github.com/etn-ccis/blui-react-native-themes?tab=readme-ov-file#typescript). 
+    - Please follow the table below for the mapping of Typography and it's replacement:
+
+<div style="overflow: auto">
+
+| Typography            | Replacement                                                           | 
```

Comment:
It will be good to mention that Material Design 2 and Material Design 3's typography structure will not be a one-to-one mapping. It is better to reach out to a UI designer (especially for those heading texts) if your project is going to be designed and developed in Material Design 3 completely.